ADR/ADR 0001-Choice of Serverless.md

```diff
@@ -46,6 +46,8 @@ We **will** prioritize the choice of serverless components of solutions before c
 
 ## Consequences
 
+![Architecture Styles Worksheet](../docs/architecture-styles-worksheet.png)
+
```

Comment:
can you add a bit more details around how our solution is a mix of the two architectures? It is a bit confusing seeing both styles.

Reply:
Sorry I will fix this a little later...

Reply:
One more thing: isn't it better to include it in the main doc, rather than the ADR?

Reply:
Since it's a worksheet that confirms that our choice of serverless matches with our architectural characteristics, i believe that it's better in the ADR

Reply:
Alright. I'll add a bit more details around the serverless microservice architecture that we chose.